WalletWasabi/Logging/Logger.cs

```diff
@@ -24,7 +24,7 @@ public static class Logger
 
 		public static string FileEntryEncryptionPassword { get; private set; } = null;
 
-		private static long _logerFailed = Interlocked.Exchange(ref _logerFailed, 0);
+		private static long _logerFailed;
```

Comment:
Could you initialize it with `=0`?

Reply:
Sure. What's the point of doing that? I mean, the initial value is always, with no exception, zero.

Reply:
Readability.